DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -97,3 +97,66 @@
 	- **Important:** select "Do not format other file types by IntelliJ formatter"
 	- go to Preferences -> Editor -> Code style -> Java : set class and static names counts for import with * to 99
 	- for Android Studio, code formatting is usually done with Ctrl+Alt+L. For automatic formatting, it's recommended to use the plugin Save Actions (https://plugins.jetbrains.com/plugin/7642-save-actions)
+
+## Enable debugging of the application
```

Comment:
Document your remarks under the headlines addressing tools to install: Everything concerning IntelliJ to be mentioned under ##IntelliJ, under a sub headline (###) if needed.